From: Barry deFreese <bdefreese@debian.org>
Origin: vendor
Bug-Debian: http://bugs.debian.org/533698
Subject: Fix build failure on Debian GNU/Hurd
 This fixes a FTBFS on Debian GNU/Hurd because it is missing a hints file.

--- /dev/null
+++ b/hints/c-gnu.h
@@ -0,0 +1,27 @@
+/*  Hint file for the GNU platform.
+ *
+ *  If this configuration doesn't work, look at the file "c-none.h"
+ *  for how to set the configuration options.
+ */
+
+#include <ncurses.h>
+
+#ifdef C_PANELSUPPORT
+#include <panel.h>
+#endif
+
+#ifdef C_MENUSUPPORT
+#include <menu.h>
+#endif
+
+#ifdef C_FORMSUPPORT
+#include <form.h>
+#endif
+
+#define C_LONGNAME
+#define C_LONG0ARGS
+#undef  C_LONG2ARGS
+
+#define C_TOUCHLINE
+#define C_TOUCH3ARGS
+#undef  C_TOUCH4ARGS
